Lombiq.HelpfulLibraries.OrchardCore/Docs/Users.md

```diff
@@ -1,19 +1,20 @@
-# User Libraries Documentation
+# Lombiq Helpful Libraries - Orchard Core Libraries - Users for Orchard Core
 
 
 
-## `ICachingUserManager`
+## `AdminPermissionBase`
 
-Retrieves `User`s from a transient per-request cache or gets them from the store if not yet cached. This is an abstraction over the `UserManager<IUser>` using its methods to retrieve the `User` from the database but caches them after the first time. Can improve performance if the `User` is retrieved multiple times per request.
+A provider that only has *Administrator* stereotype permissions. Reduces boilerplate.
 
-Please see the inline documentation of each method to learn more.
+To use it, override the `AdminPermissions` read-only abstract property with a collection of admin permissions. This base class implements the `GetPermissionsAsync` and `GetDefaultStereotypes` methods with the derived permission collection.
 
+## Extensions
 
-## `AdminPermissionBase`
+- `UserServiceExtensions`: Adds extensions for `IUserService`. (E.g. `GetOrchardUserAsync` retrieves the user by user name or throws an exception if none found.)
 
-A provider that only has *Administrator* stereotype permissions. Reduces boilerplate.
+## `ICachingUserManager`
 
-To use it, override the `AdminPermissions` read-only abstract property with a collection of admin permissions. This base class implements the `GetPermissionsAsync` and `GetDefaultStereotypes` methods with the derived permission collection.
+Retrieves `User`s from a transient per-request cache or gets them from the store if not yet cached. This is an abstraction over the `UserManager<IUser>` using its methods to retrieve the `User` from the database but caches them after the first time. Can improve performance if the `User` is retrieved multiple times per request.

```

Comment:
> transient per-request

It's the scoped service lifetime that's per-request in ASP.Net Core, not the transient. Transient is when you create a new instance for every injection. Indeed `ICachingUserManager` has a scoped lifetime (`services.TryAddScoped<ICachingUserManager, CachingUserManager>();`).

Reply:
Yep. I guess the original ide ([here](https://github.com/Lombiq/Helpful-Libraries/commit/66c3950cfff3e3d0988631d016f180d0255ced83)) was to emphasize that it's not persistent. So, I'll just write that.